### DISCUSSION TITLE
a VisiData based iDE -- Integrating Development Environment

### DISCUSSION THREAD

**Discussion by justin2004** [saulpw/visidata#2408] [Show and tell]
I use VisiData as one of the foundational tools for development around the semantic web stack (RDF/SPARQL).

You'll see how I jump from a query result in VisiData to the RDF file where the term in the VisiData cell is defined. VisiData made this really easy to implement!

https://github.com/justin2004/weblog/tree/master/iDE#readme

**Comment by saulpw** [comment]
This is really cool, @justin2004!  I love how quickly you can go from a row in VisiData to an RDF definition.  Thanks so much for recording a video!

**Reply by justin2004** [reply]
No problem! I feel like with vd and tmux it is easy to make whatever tight development feedback loop you need.
I look forward to my next tight feedback loop need so I can use this approach again. :)
